### ISSUE TITLE
open the discussion on the repo ?

### ISSUE THREAD

**Issue by fabse-hack** [Ressetkk/Vyper_SW#4]
can you open the discussion on the repo ?

i found your onshape file! is awesome! but not yet finsih?
did you finish this vyper mod ?

**Comment by Ressetkk** [comment]
Yep. It's pretty much done. Also I didn't use onshore, nor I was affiliated with them.
